subsys/bluetooth/mesh/Kconfig

```
@@ -1350,15 +1350,17 @@ config BT_MESH_USES_MBEDTLS_PSA
 	select MBEDTLS_ZEPHYR_ENTROPY
 	select MBEDTLS_PSA_CRYPTO_C
 	select MBEDTLS_USE_PSA_CRYPTO
-	select MBEDTLS_MAC_CMAC_ENABLED
-	select MBEDTLS_CIPHER_AES_ENABLED
+	select PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC
+	# Required otherwise there are missing symbols for PK helpers at link time.
```

What line(s) is this comment addressing?

---

The line below. It enables RSA keys support which might look odd since we're only using ECDH and ECDSA, so I thought adding a comment there could help. Doesn't it?

---

I see. I didn't pay much attention to what is (meant to be) used compared to what is enabled. Others can likely judge better. The comment is fine and makes sense.

---

```suggestion
	# Required, otherwise there are missing symbols for PK helpers at link time.
```
I think a comma is needed